src/core/utils/helpers/list-mapper.ts

```diff
@@ -169,4 +166,72 @@ export const mapManifestationToBasicDetailsType = (
   } as BasicDetailsType;
 };
 
+// Reservation is a reservation from Publizon, and is the equivalent
+// to the ReservationDetailsV2 type in FBS. These are mapped to the same
+// so digital/physical loans/reservations can use the same components,
+// as their UI is often quite similar
+export const mapPublizonReservationToReservationType = (
+  list: Reservation[]
+): ReservationType[] => {
+  return list.map(
+    ({
+      identifier,
+      createdDateUtc,
+      status,
+      expectedRedeemDateUtc,
+      expireDateUtc
+    }) => {
+      const publizonReservationState: { [key: number]: string } = {
+        1: "reserved", // in publizon Queued
+        2: "readyForPickup", // in publizon Redeemable
+        3: "redeemed", // in publizon Redeemed
+        4: "cancelled", // in publizon Cancelled
+        5: "expired" // in publizon Expired
+      };
+
+      return {
+        identifier,
+        faust: null,
+        dateOfReservation: createdDateUtc,
+        expiryDate: expireDateUtc,
```

Comment:
Consider converting the mentioned here to proper date/dayjs objects during mapping. This means we have much better type safety throughout the rest of the system.